lib/populate/extract-csv.js

```diff
@@ -1,7 +1,14 @@
 const {validate} = require('@ban-team/validateur-bal')
+const altLanguages = require('@ban-team/shared-data/langues-regionales.json')
 const {chain, compact, deburr, keyBy, min, max} = require('lodash')
 const {ObjectId} = require('../util/mongo')
 
+const validAltLang = new Set([])
+
+altLanguages.forEach(al => {
+  validAltLang.add('voie_nom_' + al.code)
+})
```

Comment:
Nous pourrions initialiser directement le `Set`.

```suggestion
const validAltLang = new Set(altLanguages.map(({code}) => `voie_nom_${code}`))
```